src/encoded/tests/test_renderers.py

```diff
@@ -20,8 +25,53 @@ def __init__(self, content_type=None, status_code: int = 200, json=None, content
         super().__init__(status_code=status_code, json=json, content=content, url=url)
 
 
+def test_mime_variables():
```

Comment:
Note that this and a couple other unit tests are added in this file, but importantly, the function `test_should_transform` (on [original lines 39-136](https://github.com/4dn-dcic/fourfront/pull/1588/files#diff-a8b97191f716cd1392c16aa1d416fbd43a15116d8d4ce6764a1dbfcfb139f892L39-L136)) are unchanged. This gives pretty good confidence that the core functionality being tested here is unaffected in spite of the reimplementation of some functions used therein.